sklearn/ensemble/tests/test_bagging.py

```diff
@@ -245,6 +254,16 @@ def test_bootstrap_samples():
     assert_greater(base_estimator.score(X_train, y_train),
                    ensemble.score(X_train, y_train))
 
+    # check as well the size of the training set to be sure that only
+    # under-sampling is performed
```

Comment:
I'm not sure what you mean by "only under-sampling". Do you mean to check that each is a complete bootstrap resample?

Reply:
The bug that I was introducing with the mask would apply an under-sampling of the sample instead of a bootstrapping. So you remark is correct, I will make the comment clearer.